client/context/query.go

```diff
@@ -223,7 +223,12 @@ func (ctx CLIContext) verifyProof(queryPath string, resp abci.ResponseQuery) err
 	kp = kp.AppendKey([]byte(storeName), merkle.KeyEncodingURL)
 	kp = kp.AppendKey(resp.Key, merkle.KeyEncodingURL)
 
-	err = prt.VerifyValue(resp.Proof, commit.Header.AppHash, kp.String(), resp.Value)
+	if len(resp.Value) == 0 {
```

Comment:
I would add a simple comment here above the if statement stating the two various cases.

Reply:
Comments added